service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpFileUploadNotificationReceive.java

```diff
@@ -167,14 +129,10 @@ public synchronized void onFeedbackReceived(String feedbackJson)
 
             fileUploadNotificationQueue.add(fileUploadNotification);
         }
-        catch (IOException e)
```

Comment:
Why are we removing this?

Reply:
I've refactored this a bit for the next iteration. Originally, no IOException was ever being thrown in that try block even though one was declared. In the next iteration, this will just catch any exception instead of an IOException in particular.